### PR TITLE
Add Prema28-India's pin tothe map

### PR DESCRIPTION
**Pull request by Prema28**

### CODE REVIEW

File: _pins/Prema28-India.jason [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+githubhandle: Prema28
```
Comment (hollenberry) [inline _pins/Prema28-India.jason]: Looks like githubhandle should read `githubHandle`. 👍

File: _pins/Prema28-India.jason [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+Latitude: 12.971599
```
Comment (hollenberry) [inline _pins/Prema28-India.jason]: Almost! This should be all lowercase!

File: _pins/Prema28-India.jason [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+Longitude: 77.594563
```
Comment (hollenberry) [inline _pins/Prema28-India.jason]: All lowercase here as well!

Comment (hollenberry) [inline _pins/Prema28-India.jason]: Also, don't forget to add three dashes to the bottom of your code.

**Comment by elizabethcastillo-asu** [comment]
Thank you for this helpful case @Prema28  @hollenberry

**Pull request merged**